<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connection</class>
 <widget class="QDialog" name="Connection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="DBNameLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>172</y>
     <width>287</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="RoleLabel">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>205</y>
     <width>122</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Роль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PortLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>139</y>
     <width>287</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="AcceptButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>415</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Подтвердить</string>
   </property>
  </widget>
  <widget class="QComboBox" name="RoleBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>205</y>
     <width>287</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>11</y>
     <width>415</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Параметры подключения</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="LoginLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>73</y>
     <width>287</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="HostLabel">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>40</y>
     <width>122</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Хост</string>
   </property>
  </widget>
  <widget class="QLabel" name="DBNameLabel">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>172</y>
     <width>122</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Название базы данных</string>
   </property>
  </widget>
  <widget class="QLabel" name="PortLabel">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>139</y>
     <width>122</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Порт</string>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordLabel">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>106</y>
     <width>122</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>106</y>
     <width>287</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="LoginLabel">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>73</y>
     <width>122</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Логин</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="HostLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>287</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>631</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
